### PR TITLE
append bundle script to body

### LINKED ISSUE #320: Append bundle to `doc.body`
Following PWA & PageSpeed recommendations, the `next.min.js` bundle should not be placed in the `document.head` as it delays the first paint.

Using Lighthouse on Nextgram (and zeit.co) shows that the bundle is (significantly) render-blocking, despite the pre-compiled server output.

screen shot 2016-12-01 at 6 02 53 pm

Code Responsible:

```js
function load (src, fn) {
  fn = fn || function () {}
  var script = document.createElement('script')
  script.src = src
  script.onload = function () { fn(null) }
  script.onerror = fn
  script.crossorigin = 'anonymous'
  document.head.appendChild(script)
}
```

Ofc, the last little bit can be changed to wherever, or simply appended to the `body` altogether.

```js
document.body.insertBefore(script, document.body.lastChild);
//=> div#__next
//=> <script> __NEXT_DATA__ = ...
//=> <script src='next.min.js'>
//=> <script> function load(){}

document.body.appendChild(script);
//=> div#__next
//=> <script> __NEXT_DATA__ = ...
//=> <script> function load(){}
//=> <script src='next.min.js'>
```

And since the first load is always via SSR, visual content does not rely on the bundle being loaded. Because of this, we can set `script.async = true` within the `load` function.

---

Opening issue before starting PR 👌

### PR DESCRIPTION
**Pull request by lukeed**
Closes #320.

**TL;DR:** Better first-paint performance & PageSpeed / PWA compliant.

The `commons.js` is inserted later, so the final order is:

```html
<div id="__next"></div>
<script>__NEXT_DATA</script>
<script>function load(){}</script>
<script src="next.min.js"></script>
<script src="commons.js"></script>
```

### CODE REVIEW

**Comment by arunoda** [comment]
I just tested this and looks great to me.

**Comment by lukeed** [comment]
Coveralls has been running for 3+ hours?
